src/app/search/items/item-search-filter.ts

```diff
@@ -156,6 +158,7 @@ function makeFilterContext(
   language: DimLanguage,
   customStats: Settings['customStats'],
   d2Definitions: D2ManifestDefinitions | undefined,
+  ownedItemsInfo: import('app/inventory/selectors').OwnedItemsInfo,
```

Comment:
This isn't the right way to reference a type. It should be imported as normal. That said, given the comment above suggests removing this, it doesn't need to be fixed.